hard_task: "([]<> r1) && ([]<> r2)"
soft_task: ""
